plutus-conformance/uplc/evaluation/builtin/ifThenElse-no-force/ifThenElse-no-force.uplc.expected

```diff
@@ -0,0 +1,2 @@
+An error has occurred:  A builtin received a term argument when something else was expected
```

Comment:
Input is 
```
(program 1.0.0 [ [ [ (builtin ifThenElse) (con bool True) ] (con integer 0) ] (con integer 1) ])
```
When it's `force`, like in the `ifThenElse` test, it evaluates:
```
(program 1.0.0 [ [ [ (force (builtin ifThenElse)) (con bool True) ] (con integer 0) ] (con integer 1) ])
```
Otherwise this exception is thrown. I think we talked about this before but I can't remember if this is right.

Reply:
That's what should happen, I think!